src/components/Editor/Editor.css

```diff
@@ -9,6 +9,7 @@
   --debug-line-error-border: rgb(255, 0, 0);
   --debug-expression-error-background: rgba(231, 116, 113, 0.3);
   --editor-header-height: 30px;
+  --highlight-line-duration: 1.5s;
```

Comment:
I recently noticed an issue about deleting a CSS var that was used in only one place. Perhaps keeping this inline would be nice. 🤷‍♂️